apps/condo/domains/common/constants/regexps.js

```diff
@@ -3,7 +3,7 @@ const UPPER_CASE_ALPHANUMERIC_REGEXP = /^[A-Z0-9_]+$/
 const LETTERS_AND_NUMBERS = /[\p{L}\p{N}]/gu
 const PHONE = /^\+?\d*(\.\d*)?$/
 const PHONE_CLEAR_REGEXP = /[^+0-9]/g
-const JAVASCRIPT_URL_XSS = /^[u00-u1F]*j[\r\n\t]*a[\r\n\t]*v[\r\n\t]*a[\r\n\t]*s[\r\n\t]*c[\r\n\t]*r[\r\n\t]*i[\r\n\t]*p[\r\n\t]*t[\r\n\t]*:/i
+const JAVASCRIPT_URL_XSS = /^[u00-u1F]*j[\r\n\t\s]*a[\r\n\t\s]*v[\r\n\t\s]*a[\r\n\t\s]*s[\r\n\t\s]*c[\r\n\t\s]*r[\r\n\t\s]*i[\r\n\t\s]*p[\r\n\t\s]*t[\r\n\t\s]*:/i
```

Comment:
I thought that `\s` is a shorthand for `[ \t\r\n\f]`

Reply:
Shortened expression 👍